.evergreen/config.yml

```diff
@@ -947,14 +961,25 @@ buildvariants:
   tasks:
     - name: "test-loadBalanced"
 
-- matrix_name: "test-csfle-skip_crypt_shared"
-  matrix_spec: { "os": "debian11", "mongodb-versions": "*", "php-edge-versions": "latest-stable", "driver-versions": "latest-stable" }
-  display_name: "CSFLE skip_crypt_shared - ${mongodb-versions}"
+# CSFLE tests (crypt_shared and mongocryptd) are tested on RHEL 7 as it's the only version that supports
```

Comment:
```suggestion
# CSFLE tests (crypt_shared and mongocryptd) are tested on RHEL 8 as it's the only version that supports
```

Was this a typo? The PR description said:

> This requires us to test on a platform that supports MongoDB 4.2 up to 6.0, hence the usage of RHEL 8.0 for that pipeline.

Reply:
Yes, it was left over from a previous iteration of the changes. I've updated the comment.